play-json/src/main/scala/play/api/libs/json/JsConstraints.scala

```diff
@@ -16,9 +16,12 @@ trait PathFormat {
   def at[A](path: JsPath)(implicit f: Format[A]): OFormat[A] =
     OFormat[A](Reads.at(path)(f), Writes.at(path)(f))
 
-  def nullable[A](path: JsPath)(implicit f: Format[A]): OFormat[Option[A]] =
+   def nullable[A](path: JsPath)(implicit f: Format[A]): OFormat[Option[A]] =
     OFormat(Reads.nullable(path)(f), Writes.nullable(path)(f))
 
+  def optional[A](path: JsPath, defaultValue: => Option[A])(implicit f: Format[A]): OFormat[Option[A]] =
```

Comment:
I see. I thought this way. There were 2 methods - `as` and `optional`. I've just duplicated both ot them and added defaultValue param. If you look into history you will see my previous commit was done exactly using clear naming. This added a lot of boilerplate IMO. Just check